test/org/traccar/protocol/Tk103ProtocolEncoderTest.java

```diff
@@ -8,6 +8,21 @@
 
 public class Tk103ProtocolEncoderTest extends ProtocolTest {
 
+    @Test
+    public void testEncodeOutputControl() {
+
+        Tk103ProtocolEncoder encoder = new Tk103ProtocolEncoder();
+
+        Command command = new Command();
+        command.setDeviceId(1);
+        command.setType(Command.TYPE_OUTPUT_CONTROL);
+        command.set(Command.KEY_INDEX, 0);
+        command.set(Command.KEY_DATA, "1");
```

Comment:
Is this the value we use for all other protocols?

Reply:
It's referred from the CellocatorProtocolEncoderTest.
Is anything wrong here?